test/modules/layers/test_quantization.py

```diff
@@ -121,18 +120,18 @@ def test_embed_init(self):
         )
         assert_expected(actual_weight, expected_weight)
 
-        actual_code_avg = self.vq._code_avg
+        actual_code_avg = self.vq.code_avg
         expected_code_avg = actual_weight
         assert_expected(actual_code_avg, expected_code_avg)
 
-        actual_code_usage = self.vq._code_usage
+        actual_code_usage = self.vq.code_usage
         expected_code_usage = torch.ones(self.num_embeddings)
         assert_expected(actual_code_usage, expected_code_usage)
 
     def test_ema_update(self):
```

Comment:
`test_ema_update_embedding`? Since you already updated the test name for `init_embedding`.